org.mixedrealitytoolkit.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -212,7 +212,16 @@ public void RegisterInteractor(XRBaseInteractor interactor)
             GameObject controllerObject = null;
             if (interactor is XRBaseInputInteractor controllerInteractor)
             {
-                controllerObject = controllerInteractor.xrController.gameObject;
+#pragma warning disable CS0618 // XRController is obsolete
+                if (controllerInteractor.forceDeprecatedInput)
+                {
+                    controllerObject = controllerInteractor.xrController.gameObject;
+                }
+                else
+                {
+                    controllerObject = controllerInteractor.transform.parent.gameObject;
```

Comment:
This seems a little fragile in that it requires the interactor to be a direct child of the top level "controller" object.  That may be the case for MRTK provided prefabs, but wouldn't necessarily always be the case for a custom prefab.  Would be nice if we somehow had an equivalent marker for the parent, like XRBaseController used to provide.

Reply:
@whebertML I'll be closing this PR because a [new one](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/824) was created since several changes were made while retaking this and it was easier to re-do the changes in the [new PR](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/824).  Your feedback was attended in the new PR btw, thank you for providing the feedback.